### ISSUE TITLE
add enhancements to Standards page

### ISSUE THREAD

**Issue by nicolevasilevsky** [cancerDHC/community-development#31]
https://ccdhportaldev.pedscommons.org/standardsTooling

1. Could we have a search box like on OLS? https://www.ebi.ac.uk/ols/ontologies Maybe a page with something like this OLS page? (From @gaurav) Something like https://www.w3schools.com/howto/howto_js_filter_table.asp could be pretty useful if it can be implemented in Drupal.
2. Should we add additional info to the ontologies table? Like a column with domain?
3. question - most of this text is verbatim from another site - is that okay? or should we reword it? or quote it? Or should we have a standardized description like: ontology/terminology is used to describe and classify information about X.

Related to #19

**Comment by jen-martin** [comment]
I added a search box that filters both tables at once. Let me know if there should be a separate search box for each table instead.

I added some basic styling to the Tooling table, grouping items together under the category. It makes filtering the table more complex, however. (The simplest solution is to simply keep the category rows visible at all times, which is what I ended up doing.) If folks are OK with how the search functions for this table, I can add more custom styling (for example, colors, border, fonts, spacing, etc.) if desired. Suggestions for improvements are always welcome.
